app/domain/finders/content.rb

```diff
@@ -42,13 +44,25 @@ def view
 
   def apply_filter
     scope = view[:model_name].all
+    scope = find_by_month(scope) if SpecificMonths::VALID_SPECIFIC_MONTHS.include?(@date_range)
     scope = find_by_search_term(scope) if @filter[:search_term].present?
     scope = find_by_document_type(scope) if @filter[:document_type].present?
     scope = find_by_organisation(scope)
 
     scope
   end
 
+  def find_by_month(scope)
+    month_and_year = @date_range.split("-")
```

Comment:
might be clearer and more concise to assign individual variables straight away
```suggestion
    month, year = @date_range.split("-")
```